packages/kit/src/runtime/client/renderer.js

```diff
@@ -39,6 +40,48 @@ function notifiable_store(value) {
 	return { notify, set, subscribe };
 }
 
+/**
+ * @param {any} value
+ * @param {(set: (new_value: any) => void) => any} fn
+ * @param {number | undefined} interval
+ */
+function checkable_store(value, fn, interval) {
+	const { set, update, subscribe } = writable(value);
+
+	setInterval(() => {
```

Comment:
perhaps this should only be done if `interval > 0`

Reply:
Already changed that while setting default interval to 0. Not yet pushed ;-)